extensions/rest/src/main/java/org/radargun/stages/RESTOperationsTestStage.java

```diff
@@ -25,10 +26,12 @@
  *
  * @author Martin Gencur
  */
-@Namespace(TestStage.NAMESPACE)
+@Namespace(RESTOperationsTestStage.NAMESPACE)
 @Stage(doc = "Test using RESTOperations with specific URL")
 public class RESTOperationsTestStage extends TestStage {
 
+   public static final String NAMESPACE = "urn:radargun:stages:rest:" + Version.SCHEMA_VERSION;
```

Comment:
Because the REST stage doesn't have much in common with "cache" namespace which is the default. This stage is for sending HTTP requests to a web application.